### PR TITLE
Update Nginx config with new method for FastCGI

### PR DESCRIPTION
**Pull request by polmp**
Documentation is outdated. Now it is necessary to include _fastcgi_params_ and add a param called _SCRIPT_FILENAME_.

### CODE REVIEW

**Comment by dschaper** [comment]
I don't have any way of testing this. Has `nginx` changed something that requires this new configuration? Does it work on all of our supported distributrions?

**Comment by polmp** [comment]
> I don't have any way of testing this. Has `nginx` changed something that requires this new configuration? Does it work on all of our supported distributions?

With the actual configuration I was getting a blank screen. I'm using Nginx 1.16.1 and PHP 7.2.19 (Ubuntu 18.04.3). I haven't tried it in another distribution.

File: docs/guides/nginx-configuration.md [hunk 1]
```diff
@@ -32,8 +32,9 @@ server {
         }
 
         location ~ \.php$ {
-                include snippets/fastcgi-php.conf;
+		include fastcgi_params;
```
Comment (XhmikosR) [inline docs/guides/nginx-configuration.md]: Indentation seems off; you should use the same indentation like the other block

**Comment by dschaper** [comment]
Please rebase these changes on the current master and update anything that needs it.

**Pull request merged**